tests/src/lib.rs

```diff
@@ -0,0 +1,5 @@
+#[cfg(test)]
+mod fixtures;
+
+#[cfg(test)]
+mod tests;
```

Comment:
If I put all the tests in here the compiler would complain that all the imports were unused. If I put them in their own module and marked it as a test module (I think that's what this does) then it no longer complained. LMK if there's a better way to do this ...

Reply:
Added another commit where I moved the tests from `tests/src` to `tests/tests` so you don't need to do this `#[cfg(test)]` stuff. Think this is probably better.